libs/re/Makefile

```diff
@@ -5,16 +5,15 @@
 # This is free software, licensed under the GNU General Public License v2.
 # See /LICENSE for more information.
 #
-
 include $(TOPDIR)/rules.mk
 
 PKG_NAME:=re
-PKG_VERSION:=2.0.1
-PKG_RELEASE:=1
+PKG_VERSION:=3.16.0
+PKG_RELEASE:=2
```

Comment:
On a PKG_VERSION bump the PKG_RELEASE variable should reset to 1. The PKG_RELEASE variable should increment in case that the generated package is different within single release, e.g. identified by PKG_VERSION. This also applies to baresip.

Reply:
PKG_RELEASE is reset to 1 for both.
Each package has it's own commit now

Reply:
Thank you!